src/main/scala/com/vngrs/etl/extractors/JsonExtract.scala

```diff
@@ -0,0 +1,29 @@
+package com.vngrs.etl.extractors
+
+import com.vngrs.etl.Extract
+import com.vngrs.etl.transformers.JsonToRowTransform
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+/**
+  * Extracts a JSON file(s).
+  *
+  * @param path File path(s).
+  *   In order to supply multiple files, you can use wildcards or give multiple paths separated with comma.
+  */
+final case class JsonExtract(path: String) extends Extract[Row] {
```

Comment:
Those primitive operations must be composable without creating new classes. This class is not only meaningless but also an antipattern that we must avoid as library authors.